File: packages/runner/code-examples/cpp/simple-sum.cpp
#include <iostream>

int main() {
    int a, b;

    std::cin >> a >> b;

    if (a>4) {
        std::cout << 0 << std::endl;
        return 0;
    }

    int sum = a + b;

    std::cout << sum << std::endl;

    return 0;
}

File: packages/runner/code-examples/rust/simple-sum.rs
fn main() {
    let mut buffer = String::new();
    std::io::stdin()
        .read_line(&mut buffer)
        .expect("Failed to read line");

    let nums: Vec<u32> = buffer
        .split_whitespace()
        .map(|s| s.parse::<u32>().expect("Expected a number"))
        .collect();

    let sum = nums[0] + nums[1];

    println!("{}", sum);
}
